### PR TITLE
Oracle V3 update

### PR DESCRIPTION
**Pull request by F4ever**

### CODE REVIEW

**Comment by TheDZhon** [comment]
Sorry, the proposed changes can't pass the CI due to syntax errors.
Please consider running local `yarn start` to identify the issues :pray:

File: docs/guides/oracle-operator-manual.md [hunk 1]
```diff
@@ -1,143 +1,135 @@
...
 
-To protect stETH token price from attacks with borrowed money or flash loans, `StableSwapOracle` keeps valid stETH stats from Curve Pool (ETH balance, stETH balance and stETH price), shifted by fifteen blocks, onchain. To keep these stats in actual state, the daemon reports required data to the oracle when price difference between the time-shifted stETH pool price and the last reported price exceeds the threshold limit. `StableSwapOracle` validates all of the recorded data using cryptographic proofs. The threshold limit is set in the oracle contract and is purely advisory. It's introduced to prevent sending large number transactions which are pretty costly due to the Partricia Merkle proof verification.
+- rebasing user balances,
+- distributing node operator rewards,
+- processing withdrawal requests,
+- decides which validators should initiate volunteer exit,
```
Comment (infloop) [inline docs/guides/oracle-operator-manual.md:28]: I think `making decision` could be better here

File: docs/guides/oracle-operator-manual.md [hunk 2]
```diff
@@ -1,143 +1,135 @@
...
+  --env "EXECUTION_CLIENT_URI=$EXECUTION_CLIENT_URI" \
+  --env "CONSENSUS_CLIENT_URI=$CONSENSUS_CLIENT_URI" \
+  --env "KEYS_API_URI=$KEYS_API_URI" \
+  --env "LIDO_LOCATOR_ADDRESS=$LOCATOR_ADDRESS" \
+  --env "MEMBER_PRIV_KEY=$MEMBER_PRIV_KEY" \
+  lidofinance/oracle@<image-hash> ejector
 ```
 
+**Latest image hash:**  
+`sha256:85b6f33784df5c1fd76b987f2dac5422539e5ec81a921f0e33f7c167f6e3501b`
```
[the remark this answers is not in the document]
Comment (infloop) [reply docs/guides/oracle-operator-manual.md:127]: Maybe also add example with `docker-compose` or link to docker-compose example?

[the remark this answers is not in the document]
Comment (F4ever) [reply docs/guides/oracle-operator-manual.md:127]: Fixed

File: docs/guides/oracle-operator-manual.md [hunk 3]
```diff
@@ -1,143 +1,135 @@
...
-The daemon checks the time-shifted price of stETH token in Curve ETH/stETH pool and evaluates how much this price differs from the current `StableSwapOracle` state. If the difference exceeds the threshold set in the `StableSwapOracle` contract, the daemon generates offchain proof for the new stats and sends it to the contract. The contract validates the proof and records new stETH stats onchain.
-This transaction can also fail in the case when another Lido oracle submits the updated state between the check and transaction submission.
+Full list could be found [here](https://github.com/lidofinance/lido-oracle#env-variables).
 
-#### Environment variables
+### Lido Locator address
 
-The oracle daemon requires the following environment variables:
+**Mainnet**  
+**0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb**
```
Comment (infloop) [inline docs/guides/oracle-operator-manual.md:95]: Maybe make it clickable?

File: docs/guides/oracle-operator-manual.md [hunk 4]
```diff
@@ -1,143 +1,135 @@
...
 
 1. Generate an Ethereum address.
-2. Launch and sync an Execution Layer node with JSON-RPC endpoint enabled.
-3. Launch and sync a Beacon Сhain (aka Consensus Layer) node with API endpoint enabled. (one of Lighthouse, Prysm, Teku)
-4. Launch the oracle daemon as a docker container.
-5. [**Optional**] Add alerts to Oralce's prometheus metrics. ([https://github.com/lidofinance/lido-oracle#alert-examples](Alerts examples))
-6. In case of mainnet share your address and intention to join the Oracle set with public. You need to publish it on Twitter and also write a message with a twitter link under Onboarding post on [the Research forum](https://research.lido.fi/).
-7. Propose your oracle's ethereum address to Lido Team to vote on your address being added to the Oracle Members.
+2. Launch and sync an [archive](https://ethereum.org/en/developers/docs/nodes-and-clients/#archive-node) Execution Layer node with JSON-RPC endpoint enabled.
+3. Launch and sync an [archive](https://ethereum.org/en/developers/docs/nodes-and-clients/#archive-node) Consensus Layer node with API endpoint enabled.
```
[the remark this answers is not in the document]
Comment (F4ever) [reply docs/guides/oracle-operator-manual.md:10]: Fixed

File: docs/guides/oracle-operator-manual.md (hunk 3 above)
Comment (F4ever) [reply docs/guides/oracle-operator-manual.md:95]: Fixed

File: docs/guides/oracle-operator-manual.md (hunk 1 above)
Comment (F4ever) [reply docs/guides/oracle-operator-manual.md:28]: fixed

Comment (infloop) [reply docs/guides/oracle-operator-manual.md:28]: Can't see any changes here

File: docs/guides/oracle-operator-manual.md (hunk 3 above)
Comment (infloop) [reply docs/guides/oracle-operator-manual.md:95]: I mean adding link to etherscan is a good option here

File: docs/guides/oracle-operator-manual.md [hunk 5]
```diff
@@ -1,143 +1,135 @@
 # Oracle Operator Manual
 
-This document is intended for those who wish to participate in the Lido protocol as Oracle—an entity who runs a daemon synchronizing state from Beacon Chain to Execution Layer of the protocol. To be precise, the daemon fetches the number of validators participating in the protocol, as well as their combined balance, from the Beacon Сhain and submits this data to the `LidoOracle` smart contract at the Execution Layer.
-
-The daemon also fetches historical stETH token price (shifted by fifteen blocks) from Curve ETH/stETH pool and reports any significant changes to the `StableSwapOracle` contract. Using price data from this oracle as a safeguard helps to keep stETH token price resistant to flash-loan and sandwich attacks by removing the ability to significantly change the price in a single block.
+This document is intended for those who wish to participate in the Lido protocol as Oracle - an entity who runs a daemons synchronizing state from Beacon Layer to Execution Layer of the protocol.
```
[the remark this answers is not in the document]
Comment (F4ever) [reply docs/guides/oracle-operator-manual.md]: fixed

File: docs/guides/oracle-operator-manual.md [hunk 6]
```diff
@@ -1,143 +1,135 @@
...
-3. Launch and sync a Beacon Сhain (aka Consensus Layer) node with API endpoint enabled. (one of Lighthouse, Prysm, Teku)
-4. Launch the oracle daemon as a docker container.
-5. [**Optional**] Add alerts to Oralce's prometheus metrics. ([https://github.com/lidofinance/lido-oracle#alert-examples](Alerts examples))
-6. In case of mainnet share your address and intention to join the Oracle set with public. You need to publish it on Twitter and also write a message with a twitter link under Onboarding post on [the Research forum](https://research.lido.fi/).
-7. Propose your oracle's ethereum address to Lido Team to vote on your address being added to the Oracle Members.
+2. Launch and sync an [archive](https://ethereum.org/en/developers/docs/nodes-and-clients/#archive-node) Execution Layer node with JSON-RPC endpoint enabled.
+3. Launch and sync an [archive](https://ethereum.org/en/developers/docs/nodes-and-clients/#archive-node) Consensus Layer node with API endpoint enabled.
+4. Launch and sync a [Keys API Service](https://github.com/lidofinance/lido-keys-api).
+5. Launch the accounting and ejector modules of Oracle.
+6. [**Optional**] Add alerts to Oracle's prometheus metrics.
```
[the remark this answers is not in the document]
Comment (F4ever) [reply docs/guides/oracle-operator-manual.md]: fixed

File: docs/guides/oracle-operator-manual.md [hunk 7]
```diff
@@ -1,143 +1,135 @@
...
+7. In case of mainnet share your address and intention to join the Oracle set with public. You need to publish it on Twitter and also write a message with a twitter link under Onboarding post on [the Research forum](https://research.lido.fi/).
+8. Propose your oracle's ethereum address to Lido Team to vote on your address being added to the Oracle Members.
```
[the remark this answers is not in the document]
Comment (F4ever) [reply docs/guides/oracle-operator-manual.md]: fixed

**Review by avsetsin — APPROVED** [review]

**Review by infloop — APPROVED** [review]

**Pull request merged**